doc/semitrans.xml

```diff
@@ -251,3 +251,83 @@ Transformation( [ 7, 8, 8, 7, 8, 8, 8, 7, 8, 8 ] )]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphOfActionOnPairs">
+  <ManSection>
+    <Attr Name = "DigraphOfActionOnPairs" Arg = "coll" Label = "for ever" />
+    <Oper Name = "DigraphOfActionOnPairs" Arg = "coll[, n]" Label = "never" />
+    <Returns>A digraph.</Returns>
+    <Description>
+      If <A>coll</A> is a transformation collection and <A>n</A> is a
+      non-negative integer, then <C>DigraphOfActionOnPairs(<A>coll</A>,
+        <A>n</A>)</C> returns a digraph representing the <Ref Oper="OnSets"
+        BookName="ref"/> action of the transformations in <A>coll</A> on the
+      pairs of points in <C>[1 ..  <A>n</A>]</C>.  If the optional argument
+      <A>n</A> is not specified, then by default the value of
+      <C>DegreeOfTransformationCollection(<A>coll</A>)</C> will be chosen for
+      <A>n</A>.  <P/>
+
+      The <C>DigraphOfActionOnPairs</C> has <M><A>n</A> + {n \choose 2}</M>
+      vertices: the vertices <C>[1 .. <A>n</A>]</C> correspond to the points in
```

Comment:
In the text version this thankfully gets rendered as `The  DigraphOfActionOnPairs  has n + n choose 2 vertices`